src/php/ext/grpc/php7_wrapper.h

```diff
@@ -143,8 +143,7 @@ static inline int php_grpc_zend_hash_find(HashTable *ht, char *key, int len,
 
 #define PHP_GRPC_RETURN_STRING(val, dup) RETURN_STRING(val)
 #define PHP_GRPC_MAKE_STD_ZVAL(pzv) \
-  zval _stack_zval_##pzv; \
-  pzv = &(_stack_zval_##pzv)
+  pzv = (zval *)emalloc(sizeof(zval));
```

Comment:
I believe that will be taken care of in `RETURN_DESTROY_ZVAL`. We can certainly do an audit to make sure there's no other leak.